suites/squid/rados/tier-2_rados_test-pg-dups-trimming.yaml

```diff
@@ -0,0 +1,204 @@
+# Suite is to be used to verify the dups trimming scenario on all NVMe cluster
+# Conf file to be used : conf/squid/baremetal/mero_conf.yaml
```

Comment:
Conf file for Mero cluster has not been added in conf/squid/baremetal folder. Please add the same.

Reply:
Looks like there is some rework needed here, with respect to the conf file used here. Ignore the above comment. I will take care of it with my next PR to update the conf file used.

Reply:
Noted.